### PR TITLE
fix: config.yaml in docker collector example

### PR DESCRIPTION
**Pull request by harrykimpel**

### CODE REVIEW

**Comment by jack-berg** [comment]
I don't believe this is necessary since `["env", "system" "gcp", "ec2", "azure"]` and `[env, system, gcp, ec2, azure]` are equivalent YAML.

**Comment by harrykimpel** [comment]
> system, gcp,



> I don't believe this is necessary since `["env", "system" "gcp", "ec2", "azure"]` and `[env, system, gcp, ec2, azure]` are equivalent YAML.

the issue is that there is a comma missing between system and gcp

File: other-examples/collector/docker/config.yaml [hunk 1]
```diff
@@ -53,4 +53,4 @@ service:
     metrics:
       receivers: [docker_stats]
       processors: [resourcedetection, batch]
-      exporters: [otlphttp]
\ No newline at end of file
+      exporters: [otlphttp]
```
Comment (jack-berg) [inline other-examples/collector/docker/config.yaml:56]:
```suggestion
      exporters: [otlphttp]

```

**Review by jack-berg — APPROVED** [review]

**Comment by jack-berg** [comment]
> the issue is that there is a comma missing between system and gcp

Got it! Thanks.

**Pull request merged**